src/app/components/atoms/HeartSvg/HeartSvg.tsx

```
@@ -1,39 +1,39 @@
-      />
-    </svg>
-  );
+	const HeartColor = Heart({ color: isFavorite });
+	return (
+		<svg
+			width="25"
+			height="25"
+			viewBox="0 0 122 111"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+		>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)